client/src/components/QuestionList/QuestionList.jsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 
+// This will be updated. I only added it when I was testing SearchBar
 const QuestionList = function() {
   return (
     <section>QuestionList</section>
```

Comment:
Will you be adding a component to render each individual question or are you going to handle it all in this file? I personally would recommend having a separate component that just takes care of each question based on the props handed down to it from QuestionList

Reply:
I'm thinking I'll just have QuestionList expect to take a prop of an array of questions, map over the questions and render a Question component and a related answer component for each question in the questions array.